clap_generate/tests/completions/zsh.rs

```diff
@@ -327,30 +400,26 @@ esac
 (( $+functions[_my_app_commands] )) ||
 _my_app_commands() {
     local commands; commands=(
-        "second:" \
-"help:Prints this message or the help of the given subcommand(s)" \
+'second:' \
+'help:Prints this message or the help of the given subcommand(s)' \
     )
     _describe -t commands 'my_app commands' commands "$@"
 }
 (( $+functions[_my_app__help_commands] )) ||
 _my_app__help_commands() {
-    local commands; commands=(
-        
-    )
+    local commands; commands=()
     _describe -t commands 'my_app help commands' commands "$@"
 }
 (( $+functions[_my_app__second_commands] )) ||
 _my_app__second_commands() {
     local commands; commands=(
-        "third:" \
+'third:' \
```

Comment:
Don't we need to indent these?

Reply:
They don't need indentation, even though it would be nice to have it.

However, the current solution only indented the first line and ignored all following lines, which looked rather weird and inconsistent.

I'm not sure what's the best way to properly handle indentation in a generic way. This is probably something that should be done in it's own PR.